docs/workflows/sign-images.rst

```diff
@@ -0,0 +1,301 @@
+=================================================================================
+
+It is possible to sign content that was syncronized from remote registries.
+If content was synced together with signatures, upon signing task completion new signatures will be
+added but original ones will be kept::
+
+       $ http https://pulp.example.com/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/
+
+        {
+            "base_version": null,
+            "content_summary": {
+                "added": {
+                    "container.blob": {
+                        "count": 9,
+                        "href": "/pulp/api/v3/content/container/blobs/?repository_version_added=/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/"
+                    },
+                    "container.manifest": {
+                        "count": 5,
+                        "href": "/pulp/api/v3/content/container/manifests/?repository_version_added=/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/"
+                    },
+                    "container.tag": {
+                        "count": 2,
+                        "href": "/pulp/api/v3/content/container/tags/?repository_version_added=/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/"
+                    }
+                },
+                "present": {
+                    "container.blob": {
+                        "count": 9,
+                        "href": "/pulp/api/v3/content/container/blobs/?repository_version=/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/"
+                    },
+                    "container.manifest": {
+                        "count": 5,
+                        "href": "/pulp/api/v3/content/container/manifests/?repository_version=/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/"
+                    },
+                    "container.tag": {
+                        "count": 2,
+                        "href": "/pulp/api/v3/content/container/tags/?repository_version=/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/"
+                    }
+                },
+                "removed": {}
+            },
+            "number": 1,
+            "pulp_created": "2022-01-04T19:23:03.899602Z",
+            "pulp_href": "/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/versions/1/",
+            "repository": "/pulp/api/v3/repositories/container/container/2629ca48-1d98-4ce1-88f2-accf2de9de95/"
+        }
+
+In order to ahdere to the `container signature specs <https://github.com/containers/image/blob/main/docs/containers-signature.5.md>`_,
+``future_base_path`` needs to be provided to the sign call. This information will be used in the
```

Comment:
Do we need to require it for push repositories? I thought we only need it for the synced ones.
To keep the same interface, I would allow to pass it even when signing the push repo but fail if it does not match its distribution path. What do you think?

Reply:
Ignore me, it's in the section for synced repos. All good.